packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -48,6 +51,28 @@ typedef SetTextHandler = void Function(String text);
 /// Returned by [SemanticsConfiguration.getActionHandler].
 typedef SemanticsActionHandler = void Function(Object? args);
 
+/// The semantics merge rule generated from [SemanticsConfiguration.merger].
+class SemanticsMergeRule {
```

Comment:
The name "Rule" is a little odd. It's not really defining a rule, isn't it more the result of calling the merger?